




<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

<!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Purchasing an mtDNA test | Hank de Wit</title>
<meta name="generator" content="Jekyll v3.9.3" />
<meta property="og:title" content="Purchasing an mtDNA test" />
<meta name="author" content="Hank de Wit" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="What can you expect from an mtDNA test?" />
<meta property="og:description" content="What can you expect from an mtDNA test?" />
<link rel="canonical" href="http://localhost:4000/mtdna/dna/2021/01/08/Purchasing-an-mtDNA-test.html" />
<meta property="og:url" content="http://localhost:4000/mtdna/dna/2021/01/08/Purchasing-an-mtDNA-test.html" />
<meta property="og:site_name" content="Hank de Wit" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2021-01-08T00:00:00+10:30" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Purchasing an mtDNA test" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"BlogPosting","author":{"@type":"Person","name":"Hank de Wit"},"dateModified":"2021-01-08T00:00:00+10:30","datePublished":"2021-01-08T00:00:00+10:30","description":"What can you expect from an mtDNA test?","headline":"Purchasing an mtDNA test","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/mtdna/dna/2021/01/08/Purchasing-an-mtDNA-test.html"},"publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/img/logo.png"},"name":"Hank de Wit"},"url":"http://localhost:4000/mtdna/dna/2021/01/08/Purchasing-an-mtDNA-test.html"}</script>
<!-- End Jekyll SEO tag -->

    <link rel="stylesheet" href="/assets/css/colors-auto.css?v=">
    <link rel="stylesheet" href="/assets/css/style.css?v=">

    
      <link rel="preload" href="/assets/img/logo.png" as="image">
    

    <!-- Setup Google Analytics -->
<!-- 
 -->

<!-- You can set your favicon here -->

  <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">


<!-- <link rel="preload" href="/assets/img/wordcloud.svg" as="image"> -->

<!-- start custom head snippets, customize with your own _includes/head-custom.html file -->
<!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free/css/all.min.css"> -->

<!-- end custom head snippets -->

  </head>
  <body>
    <div class="wrapper">
      <div class="sidebar">
        <header>
          
            
              <img class="img-circle" src="/assets/img/logo.png" alt="Logo">
            
          

          
            <h1>Hank de Wit</h1>
          

          
            <p class="addr"><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="img" style="vertical-align: -0.125em;" width="1em" height="1em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"><g fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"><path d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"/><path d="m22 6l-10 7L2 6"/></g></svg>&nbsp;hdewit at people net au</p>
          

          <!-- <p></p> -->
          <p>Retired Meteorologist/IT Administrator @ Bureau of Meteorology, Australia.</p>

          <div class="link-wrapper" style="margin-bottom: 20px">
    <style scoped>
      ul.link a {
        color:var(--clr-h1-and-bold);
        text-decoration:none;
      }

      ul.link a:hover, a:focus {
        color: var(--clr-a-text-hvr);
      }
    </style>
    <ul class="link">
      <!-- <li><a href="/"><i class="<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="currentColor" role="img" style="vertical-align:-0.125em;" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"> <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0zM4.5 7.5a.5.5 0 0 0 0 1h5.793l-2.147 2.146a.5.5 0 0 0 .708.708l3-3a.5.5 0 0 0 0-.708l-3-3a.5.5 0 1 0-.708.708L10.293 7.5H4.5z"/> </svg>"></i> Home</a></li> -->
       <li><a href="/"><svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="currentColor" role="img" style="vertical-align:-0.125em;" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"> <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0zM4.5 7.5a.5.5 0 0 0 0 1h5.793l-2.147 2.146a.5.5 0 0 0 .708.708l3-3a.5.5 0 0 0 0-.708l-3-3a.5.5 0 1 0-.708.708L10.293 7.5H4.5z"/> </svg>&nbsp;Home</a></li>
      <!-- <li><a href="/mtdna"><i class="<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="currentColor" role="img" style="vertical-align:-0.125em;" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"> <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0zM4.5 7.5a.5.5 0 0 0 0 1h5.793l-2.147 2.146a.5.5 0 0 0 .708.708l3-3a.5.5 0 0 0 0-.708l-3-3a.5.5 0 1 0-.708.708L10.293 7.5H4.5z"/> </svg>"></i> mtDNA</a></li> -->
       <li><a href="/mtdna"><svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="currentColor" role="img" style="vertical-align:-0.125em;" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"> <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0zM4.5 7.5a.5.5 0 0 0 0 1h5.793l-2.147 2.146a.5.5 0 0 0 .708.708l3-3a.5.5 0 0 0 0-.708l-3-3a.5.5 0 1 0-.708.708L10.293 7.5H4.5z"/> </svg>&nbsp;mtDNA</a></li>
      <!-- <li><a href="/cfs_map/geographic_converter.shtml"><i class="<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="currentColor" role="img" style="vertical-align:-0.125em;" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"> <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0zM4.5 7.5a.5.5 0 0 0 0 1h5.793l-2.147 2.146a.5.5 0 0 0 .708.708l3-3a.5.5 0 0 0 0-.708l-3-3a.5.5 0 1 0-.708.708L10.293 7.5H4.5z"/> </svg>"></i> CFS Maps</a></li> -->
       <li><a href="/cfs_map/geographic_converter.shtml"><svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="currentColor" role="img" style="vertical-align:-0.125em;" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"> <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0zM4.5 7.5a.5.5 0 0 0 0 1h5.793l-2.147 2.146a.5.5 0 0 0 .708.708l3-3a.5.5 0 0 0 0-.708l-3-3a.5.5 0 1 0-.708.708L10.293 7.5H4.5z"/> </svg>&nbsp;CFS Maps</a></li>
    </ul>
  </div>

        </header>

        <div class="link-wrapper-mobile" id="link-wrapper-mobile">
    <!-- <style scoped>
        #link-wrapper-mobile a {
        color:var(--clr-h1-and-bold);
        text-decoration:none;
        }

        #link-wrapper-mobile a:hover, a:focus {
        color: var(--clr-a-text-hvr);
        }
    </style> -->
    <nobr><a href="/" rel="me">Home</a> / </nobr>
    <nobr><a href="/mtdna" rel="me">mtDNA</a> / </nobr>
    <nobr><a href="/cfs_map/geographic_converter.shtml" rel="me">CFS Maps</a></nobr>
</div>


        <div class="sidebar-footer"><p><small>Powered by <a href="https://jekyllrb.com/">Jekyll</a> & <a href="https://www.bodunhu.com/minimalist/">Minimalist</a></small></p>
</div>
      </div>
      <section>

      <small>8 January 2021</small>
<h1>Purchasing an mtDNA test</h1>

<p class="view">by Hank de Wit</p>

<h2 id="what-can-you-expect-from-an-mtdna-test">What can you expect from an mtDNA test?</h2>

<h3 id="haplogroup">Haplogroup</h3>

<p>The most basic piece of information you should be able to get from your mtDNA test is your Haplogroup. Note that there is also a YDNA Haplogroup which uses a similar naming convention so people often confuse the two. This test is for your direct female line ancestry and will reveal your female line Haplogroup. The Haplogroup should provide strong hints about the origin of this line.</p>

<p>The main named Haplogroups are generally very old, <strong>H</strong> for example, is about 18,000 years old and probably originated in hunter-gather group in the Near East, probably in today’s Turkey. The parent Haplogroup for <strong>H</strong> is <strong>HV</strong>. <strong>H</strong> should really just be called <strong>HV22</strong> (or similar) since it is just one of the twenty something branches of <strong>HV</strong>. But, because it was a very successful branch, it has been given it’s own name. <strong>H</strong> was quite rare in the Neolithic, apart from <strong>H1</strong>. Something happened in the early Bronze Age when <strong>H</strong> became very common in Europe. However most of the <strong>H</strong> branches are older than this event which suggests a long stable, rather isolated, population suddenly expanding. Further research on ancient DNA might one day provide an explanation.</p>

<p>Haplogroups are not markers for ethnicity as they are even more basic. A woman carrying a particular Haplogroup could join a new ethnic community and within several generations all her female offspring will be indistinguishable from others within that community, but they will still carry the original woman’s Haplogroup. This can happen time and time again. A few extra mutations will appear on some of the descendant lines, but the Haplogroup persists, and may survive today in quite a different ethnic population than the original woman.</p>

<h3 id="extra-mutations">Extra Mutations</h3>

<p>To get any real benefit from your mtDNA test you really need a full-sequence test. With a full sequence test you will not only get a more accurate assessment of Haplogroup, but you will get a list of mutations where you differ from the Haplogroup. These mutations would generally be younger than the Haplogroup thus providing more recent “bread-crumbs” along your ancestral line.</p>

<p>If enough people in your Haplogroup share these mutations it may indicate a new branch. If there are a number of mutations and matches you might be able to figure out the time-order of the mutations. The presence and extent of sub-branching could provide clues about the age of these extra mutations. This is the case with H37. It is unrecognised by the existing officiel phylotree, but has four and more extra mutations and showing quite distinct extra branches.</p>

<h3 id="matches">Matches</h3>

<p>Probably the most useful nuggets of information comes from matches. Only two DNA companies provide matching services, <a href="https://www.yfull.com/orderm/">YFull</a>, and <a href="https://www.familytreedna.com/products/mt-dna">FTDNA</a>. Basically you will be given information about how close you are to other testers, and some ability to contact them. I think the most useful matching service is through FTDNA. You will be given match information for people who match you within three mutation differences. Your closest possible match would have zero genetic differences (mutation differences) with you. I’ll refer to these as GD0 matches. If you have no GD0 differences check your mutations.</p>

<p>If one or more of your mutations end with a letter that is not an A, C, G, or T your sample may have a <a href="https://help.familytreedna.com/hc/en-us/articles/4742569461263-Understanding-mtDNA-Heteroplasmy#heteroplasmy-0-2">heteroplasmy</a> . Heteroplasmy’s are not generally useful for genealogy, and would normally just be ignored as extra mutations when comparing with your matches, but FTDNA do count this a ONE difference in their genetic difference computation. In that case your GD1 matches should be counted as if they were GD0 matches.</p>

<p>If your matches have included their most distant female line ancestor your might be able to determine a region or country where your female line originated. GD3 matches are the most distant from you, but could still proved useful. Don’t ignore them.</p>

<p>Never-the-less mtDNA mutations are typically slow so even one mutation difference could mean thousands of years of separation. Don’t expect a close relationship even with a GD0 match. If you do share autosomal DNA as well, that would be a good indicator of a close match. I have four GD0 matches. They are all North American with French origins. My ancestral line likely did also come from France, but not before 1600. The common ancestor was likely centuries earlier at best.</p>

<h4 id="how-many-matches-is-good">How many matches is good?</h4>

<p>Most times more is better, but in the mtDNA world, you can have too many. While it is a very good thing to have up to 10 or so GD0 matches, if you have 20 or more, then it is likely that your mtDNA is very old. It is very likely that you don’t have any extra mutations. For example, many people are just <strong>H</strong> and have no extra mutations. They have hundreds of GD0 matches. <strong>H</strong> is about 18,000 years old. Without any mutations to separate them we cannot ever determine their subsequent migration history. It seems incredible that there could be people today that have an identical mtDNA to their ancestor from such a remote period, but there are hundreds in the FTDNA database and probably millions untested.</p>

<p>So ideally you’d like a handful of GD0 matches. If they have a common origin then you are very lucky, unless they are closely related. Some people test their entire family, so their exact mtDNA (termed haplotype) will appear multiple times in the database. They are not helpful for genealogy as we would prefer a random sample of unrelated matches.</p>

<h2 id="where-can-get-an-mtdna-test">Where can get an mtDNA Test?</h2>

<h3 id="yseq">YSEQ</h3>

<p>You can purchase an <a href="https://www.yseq.net/product_info.php?cPath=28&amp;products_id=38291&amp;osCsid=46f722a4ee1facc677c4c4839f0131bb">mtComplete</a> test at YSEQ but there is no <em>matching service</em> at YSEQ. However, you can then upload your resulting FASTA file to <a href="https://www.yfull.com/orderm/">YFull</a> for an extra $US25 to get a matching service and have YFull add you anonymously to their <a href="https://www.yfull.com/mtree/H37/">MTree</a>.</p>

<h3 id="ftdna">FTDNA</h3>

<p>Currently the best mtDNA test is with <a href="https://www.familytreedna.com/products/mt-dna">Family Tree DNA (FTDNA)</a> where they will match your results with other tests on their database that have a difference of less than four mutations with you. You will be added anonymously to their public <a href="https://www.familytreedna.com/public/mt-dna-haplotree/H">MTree</a>. FTDNA currently have the biggest database of mtDNA samples to match against.</p>

<p>You can also subsequently upload your FASTA file to <a href="https://www.yfull.com/orderm/">YFull</a> to get the benefits of their interpretive service. I would encourage this as YFull are still the only company adding new branches to the mtDNA phylotree. For the <strong>H37</strong> Haplogroup this would improved the dating of <strong>H37</strong> in the [MTree]( https://www.yfull.com/mtree/H37/.</p>

<h3 id="23me">23&amp;me</h3>

<p>The mtDNA test at <a href="https://customercare.23andme.com/hc/en-us/articles/212880257-Maternal-Haplogroups-mtDNA">23&amp;me</a> is not a full-sequence test. 23&amp;me test a selection of positions (SNPs) of the mt-chromosome and use this to predict your Haplogroup. The predictions are fairly accurate for the most part. However 23&amp;me do not use the latest phylotree for their Haplogroup assignment, and this can cause them to assign Haplogroups that have changed or even been deleted. For example, in my own case the 2011 version of the phylotree had a <strong>H37</strong> Haplogroup, but this was later dropped and no <strong>H37</strong> exists on the official <a href="https://www.phylotree.org/">phylotree</a>. 23&amp;me cannot provide you with your extra mutations, which would be of mot ue to you in family tree research. Haplogroups alone are of little use. 23&amp;me also can’t provide you with any matches.</p>



  <small>tags: <em></em></small>



      </section>
      <footer><p><small>Powered by <a href="https://jekyllrb.com/">Jekyll</a> & <a href="https://www.bodunhu.com/minimalist/">Minimalist</a></small></p>
</footer>
    </div>
    
      <script src="/assets/js/scale.fix.js"></script>
    
  </body>
</html>
